runtime/doc/editing.txt

```diff
@@ -1236,10 +1243,19 @@ present in 'cpoptions' and "!" is not used in the command.
 							*:chd* *:chdir*
 :chd[ir][!] [path]	Same as |:cd|.
 
+							*:tc* *:tcd*
+:tc[d][!] {path}        Like |:cd|, but set the current directory for the
+                        current tab.  The current directory for other tabs
+                        or the local directory of windows in this tab is not
+                        changed, except for the current window.
+
+							*:tch* *:tchdir*
+:tch[dir][!]		Same as |:tcd|.
+
```

Comment:
`:lcd -` existed but wasn't mentioned. I think if it exists we should mention it.